### ISSUE TITLE
[5.1] Menus issue on mobile

### ISSUE THREAD

**Issue by alecpl** [tinymce/tinymce#5232]
![Screenshot_20191104_081134](https://user-images.githubusercontent.com/546788/68104426-47f45a80-fedb-11e9-8c71-e4c115732316.png)

What happened to the mobile menu known from v5.0? Or maybe mobile device detection is not working? It is like this in Chrome mobile mode as well as on my iOS 12.4.1 device.

**Comment by lnewson** [comment]
The wrapping possibly needs investigating and could be improved, but the display is correct as 5.1 will use the silver theme by default now, instead of the older mobile theme as noted in our [blog post](https://go.tiny.cloud/blog/update-changes-to-default-ui-configuration-in-tinymce-5-1/) and [changelogs](https://www.tiny.cloud/docs/changelog/#version510october172019). If you want the legacy mobile mode, then you'd need to add the following to your init config.
```
mobile: { theme: 'mobile' }
```

**Comment by alecpl** [comment]
Ok. Does that mean that the mobile theme is deprecated? Can we expect fixes in that realm? I'd like to know the future of the mobile theme before I start implementing workarounds. Imo, we indeed don't need the mobile theme if the navigation is improved. From my perspective only this one menu needs fixing.

**Comment by lnewson** [comment]
Yes that's correct, the old theme will be deprecated and we'll likely only be making critical fixes for the old mobile theme moving forwards. We won't be removing the old theme anytime soon though.

**Comment by lostkeys** [comment]
Do you have huge headings in your document? AFAIK the styles for the menu is fetched from the content. IMO it's still something we need to look at, I just want to narrow down the issue. Just slapping on a `whitespace: no-wrap` probably won't solve your issue either as the menu would become too wide.

**Comment by alecpl** [comment]
Indeed, tinymce body was set to 18pt while testing. So, it might not be the best live scenario. However, it shows that this menu might not be the best solution for mobile.
